package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "redoc",
-  "version": "2.0.0-rc.65",
+  "name": "redoc-etn",
```

Comment:
note: i should have created a separate branch for these changes, but I made them on the same branch for the PR. This change was just so that I could run an `npm install -g redoc-cli-etn` that would use the version of redoc that includes `x-badges` support